frontend/src/metabase/public/components/EmbedFrame/useGlobalTheme.tsx

```diff
@@ -0,0 +1,26 @@
+import { useEffect } from "react";
```

Comment:
where is this file used?

Reply:
@WiNloSt Good catch, this file shouldn't exist yet in this timeline. Removed.